spring.cloud.gcp.sql.database-name=la-mysql
spring.cloud.gcp.sql.instance-connection-name=newagent-25039:asia-northeast1:la-mysql
#spring.datasource.username=postgres
spring.datasource.password=la123456
# So app starts despite "table already exists" errors.
spring.datasource.continue-on-error=true

#spring.cloud.gcp.project-id=
#spring.cloud.gcp.credentials.location=file:/path/to/service-account.json
